### ISSUE TITLE
grpc error adding hunt cell to notebook

### ISSUE THREAD

**Issue by lukerosser** [Velocidex/velociraptor#3775]
When trying to "Add cell from hunt" to a notebook, it's unable to load the list of hunts to choose from - the number of hunts is a factor but we have more in some organisations so I'm not sure exactly what's bumping the size

```
Error: grpc: received message larger than max (4875804 vs. 4194304)
```

**Comment by scudette** [comment]
Can you please check which api call is failing? Open the dev console in the browser and click the network tab to see which call is red

**Comment by lukerosser** [comment]
Request URL: https://<velociraptor>/api/v1/ListHunts?count=100&offset=0
Status Code: 429 Too Many Requests

**Comment by scudette** [comment]
Thanks for reporting this. It should be fixed with #3780 which removed unneeded data from the API call so should help to fit it in the limit
